lib/yu.rb

```diff
@@ -250,7 +250,10 @@ def template_dir
       relative_path.realpath
     end
 
-    def run_command(command, showing_output: true, exit_on_failure: true)
+    def run_command(command, options={})
```

Comment:
yes, you are right; I was so overeager building backwards compatibility that I pushed changes which would make it look like 1.9.x :)